### PR TITLE
(WIP) Research: light dismiss

### PR DESCRIPTION
**Pull request by erikkroes**
Following up on https://github.com/openui/open-ui/issues/325

Researching what a light dismiss could be.

### CODE REVIEW

File: research/src/pages/light-dismiss/light-dismiss.research.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Checkbox Concepts
```
Comment (gregwhitworth) [inline research/src/pages/light-dismiss/light-dismiss.research.mdx:2]: Should be Light Dismiss

**Review by gregwhitworth — COMMENTED** [review]
I appreciate that this is a WIP PR, so maybe switch it to a draft to alleviate reviews prior to you being ready? https://github.blog/2019-02-14-introducing-draft-pull-requests/

@melanierichards do you have research that should be joined in with this?

File: research/src/pages/light-dismiss/light-dismiss.research.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+path: /components/light-dismiss.research
```
Comment (gregwhitworth) [inline research/src/pages/light-dismiss/light-dismiss.research.mdx:3]: I'm curious if this should sit on its own right now? Maybe it should be a part of `<popup>`?

File: research/src/pages/light-dismiss/light-dismiss.research.mdx (hunk 1 above)
Comment (erikkroes) [reply research/src/pages/light-dismiss/light-dismiss.research.mdx:2]: Oh, I didn't update the file when copy pasting. Sorry about that!

File: research/src/pages/light-dismiss/light-dismiss.research.mdx (hunk 2 above)
Comment (erikkroes) [reply research/src/pages/light-dismiss/light-dismiss.research.mdx:3]: My idea is that it's an independant concept that applies to `<popup>` but can imagine it "loops back" at some point.

**Comment by erikkroes** [comment]
> I appreciate that this is a WIP PR, so maybe switch it to a draft to alleviate reviews prior to you being ready? https://github.blog/2019-02-14-introducing-draft-pull-requests/
> 
> @melanierichards do you have research that should be joined in with this?

I'll bump this up on my todolist. Hopefully update it further soon.

**Comment by gregwhitworth** [comment]
@erikkroes where are we at with this, should we close this out and you can re-open this when you get time to focus on it?

**Comment by gregwhitworth** [comment]
Closing this due to in-activity, we can always re-open this in the future.

**Pull request closed without merging**